java-squid/src/main/java/org/sonar/java/se/ProgramState.java

```diff
@@ -68,10 +70,12 @@ public Pop(ProgramState programState, List<SymbolicValue> result) {
 
   private final Deque<SymbolicValue> stack;
   private final PMap<Symbol, SymbolicValue> values;
+  private final PMap<SymbolicValue, Integer> references;
   private final PMap<SymbolicValue, Object> constraints;
-  private ProgramState(PMap<Symbol, SymbolicValue> values, PMap<SymbolicValue, Object> constraints, PMap<ExplodedGraph.ProgramPoint, Integer> visitedPoints,
+  private ProgramState(PMap<Symbol, SymbolicValue> values, PMap<SymbolicValue, Integer> references, PMap<SymbolicValue, Object> constraints, PMap<ExplodedGraph.ProgramPoint, Integer> visitedPoints,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 197 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00103)